.github/workflows/ci-polaris-react.yml

```diff
@@ -10,36 +10,6 @@ on:
       - 'polaris-react/**'
```

Comment:
```suggestion
      - 'polaris-react/**'
```
Should this run without the filter? The file could also be renamed?!

Reply:
Probably so, since a change in other packages might cause a change in the Viz and A11y tests. Good callout

Reply:
At this point, where there's no filtering here and the `on` logic is the same as ci.yml, could these be jobs in the ci.yml file?

Reply:
Decided to keep the filter and CI separate since jobs can't be filtered based on path. Only full workflows can be triggered conditionally by path.